nuxt.config.js

```diff
@@ -4,11 +4,9 @@ import fs from "fs";
 // eslint-disable-next-line no-undef
 let local = process.env.NUXT_ENV_LOCAL !== undefined;
 
-const baseUrl = // eslint-disable-next-line no-undef
-(process.env.BASE_URL || "https://www.whatsanalyze.com").replace(
-  "http:",
-  "https:"
-);
+const baseUrl = ( // eslint-disable-next-line no-undef
```

Comment:
you did not change anything in this file, right?
I dont understandy why the husky ci does not fail

Reply:
ah das war höchstens ausversehen...

Reply:
ne, das steht da immer noch, kp warum das GitHub so zeigt:

das ist nur umformattiert

```
const baseUrl = ( // eslint-disable-next-line no-undef
  process.env.BASE_URL || "https://www.whatsanalyze.com"
).replace("http:", "https:");
```

Reply:
Könnt ihr das nicht einfach reversen. 
Sonst macht prettify das beim nächsten wieder anders und man muss es wieder megen.

Ansonsten passt es.